### ISSUE TITLE
 Error: "ArgumentError : wrong number of arguments (given 1, expected 0)"

### ISSUE THREAD

**Issue by hanhdt** [lebedev-yury/carrierwave-base64#56]
Hi all,
After update gem version from 2.5.0 to 2.5.1.1 doesn't work uploading base64.

My stack:
```
rails 5.0.3
carrierwave 0.11.2
carrierwave-base64 2.5.1.1
```

In model:
```
class Profile < ApplicationRecord
  mount_base64_uploader :picture, ImageUploader
end
```
when I try PUT update profile with picture base64 string in format: 
`data:image/png;base64,` <Base64Encoded data>`

I get error wrong number of arguments (given 1, expected 0)

Do you have any ideas what's wrong?

Thanks.

**Comment by szajbus** [comment]
I just submitted a PR with a fix. 

Until it gets merged, the workaround is to pass a correct proc in `options`:

```ruby
mount_base64_uploader :image, ImageUploader, file_name: -> (_) { "image" }
```

**Comment by hanhdt** [comment]
Yes, 

I already test your workaround. It worked for me, just update to generate random filename that avoid get stale data:
```
require 'securerandom'
class Profile < ApplicationRecord
  mount_base64_uploader :picture, ImageUploader, file_name: -> (_) { "picture" + SecureRandom.hex }
end
```
Thanks you a lot.

**Comment by lebedev-yury** [comment]
Also, if you don't want to pass `_` to the lambda, you can use `Proc.new`, it doesn't check the number of arguments

**Comment by lebedev-yury** [comment]
The issue is fixed in 2.5.2
